### PR TITLE
FC-0057: Updated README with the installation guide

### PR DESCRIPTION
**Pull request by Ali-Salman29**
Updated README with installation steps, development guide, and project motivation.

close https://github.com/edly-io/forum/pull/3

### CODE REVIEW

**Review by taimoor-ahmed-1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,86 @@
...
+## Installation Steps
+
+Follow the steps below to install the Forum app in your Tutor environment:
+
+### 1. Add a Forum App to the openedX dependencies
+
+Open your Tutor configuration file, typically located at `~/.tutor/config.yml`. Add the following package in the `OPENEDX_EXTRA_PIP_REQUIREMENTS`. You can also access the config file via `vim "$(tutor config printroot)/config.yml"` . We are installing the package from Git because the package isn’t yet available on PyPi.
+
+```bash
+OPENEDX_EXTRA_PIP_REQUIREMENTS = [
```
Comment (regisb) [inline README.md:22]: It's better to not edit the file manually, and to run instead: 

    tutor config save --append 'OPENEDX_EXTRA_PIP_REQUIREMENTS=git+https://github.com/edly-io/forum.git@master'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,86 @@
...
+tutor images build openedx
+```
+
+### 3. Run Launch
+
+Run the launch based on the environment that you are using. The launch is necessary to build the migrations present in the Forum app. 
+
+```bash
+# Dev
+tutor dev launch
```
Comment (regisb) [inline README.md:40]: Let's only cover "dev".

File: README.md [hunk 3]
```diff
@@ -1,2 +1,86 @@
...
+
+1. **Create the Plugin File:**
+Create a file named `forumv2.py` inside the `tutor-plugins` folder. You can navigate to this folder using the command: `$(tutor plugins printroot)`.
+2. **Insert the Following Code:**
+Paste the following code into `forumv2.py`. Tutor consider this file as a plugin and you can verify it via `tutor plugins list` command. 
+    
+    ```python
+    from tutor import hooks
+    
+    hooks.Filters.MOUNTED_DIRECTORIES.add_item(("openedx", "forum"))
```
Comment (regisb) [inline README.md:65]: Let's not do that. Instead, follow the instructions from this plugin: https://gist.github.com/taimoor-ahmed-1/9e947a06d127498a328475877e41d7c0

File: README.md [hunk 4]
```diff
@@ -1,2 +1,91 @@
...
+    ```
+
+7. **Launch the platorm:**
+    ```bash
+    tutor dev launch
+    ```
+
+## License
+
+This project is licensed under the MIT License.
```
Comment (Faraz32123) [inline README.md:91]: You can also copy these datails from [README.rst](https://github.com/edly-io/forum/blob/master/README.rst#:~:text=Getting%20Help%20page.-,License,are%20expected%20to%20follow%20the%20Open%20edX%20Code%20of%20Conduct.,-People).

Comment (Ali-Salman29) [reply README.md:91]: I think there should be only one README file. It's better to delete any one of it.

Comment (Faraz32123) [reply README.md:91]: Yes, we can delete one.

And, we already have a [LICENSE.txt](https://github.com/edly-io/forum/blob/master/LICENSE.txt). You can also link that in the readme. Contributing and The Open edX Code of Conduct can also be added.

**Review by Faraz32123 — APPROVED** [review]
LGTM!

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Comment by regisb** [comment]
Let's add a note that we should be running this fork of edx-platform: https://github.com/edly-io/edx-platform/pull/571

**Comment by regisb** [comment]
Also, maybe we should explain how to log all mongodb queries, such that we can troubleshoot more easily how forum v1 fetches and updates data? 
https://www.mongodb.com/docs/manual/reference/log-messages/
https://stackoverflow.com/questions/15204341/mongodb-logging-all-queries

**Pull request merged**